datadotworld/client/api.py

```diff
@@ -714,7 +714,13 @@ def sql(self, dataset_key, query, desired_mimetype="application/json", **kwargs)
             header_value='Bearer {}'.format(self._config.auth_token),
             default_mimetype=desired_mimetype)
         sql_api_client.user_agent = _user_agent()
-        sql_api = _swagger.SqlApi(sql_api_client)
+
+        sql_api = None
```

Comment:
A more idiomatic way of doing this would be:
`sql_api = kwargs.get('sql_api_mock', _swagger.SqlApi(sql_api_client))`